### PR TITLE
MOBILE-4676: Use core strings

### PR DESCRIPTION
**Pull request by jleyva**

### CODE REVIEW

File: scripts/langindex.json [hunk 1]
```diff
@@ -326,10 +326,10 @@
   "addon.messages.selfconversationdefaultmessage": "message",
   "addon.messages.sendcontactrequest": "message",
   "addon.messages.showdeletemessages": "local_moodlemobileapp",
-  "addon.messages.type_blocked": "local_moodlemobileapp",
-  "addon.messages.type_offline": "local_moodlemobileapp",
-  "addon.messages.type_online": "local_moodlemobileapp",
-  "addon.messages.type_search": "local_moodlemobileapp",
+  "addon.messages.type_blocked": "message/blockedusers",
+  "addon.messages.type_offline": "message",
```
Comment (crazyserver) [inline scripts/langindex.json:330]: type_offline does not exist, use 'message/offline' instead.

File: scripts/langindex.json [hunk 2]
```diff
@@ -326,10 +326,10 @@
...
+  "addon.messages.type_online": "message",
```
Comment (crazyserver) [inline scripts/langindex.json:331]: type_online does not exist, use 'message/online' instead.

**Review by crazyserver — CHANGES_REQUESTED** [review]

**Comment by crazyserver** [comment]
Additional to that, please change strings on lang.json from
addon.block_myoverview.nocoursesenrolled to 'You are currently not enrolled in any courses.'
addon.mod_glossary.searchquery to 'Enter your search query'
addon.messages.type_blocked to 'Blocked users'

I think the other ones do not change

**Comment by jleyva** [comment]
> i

@crazyserver I just applied the requested changes

**Comment by crazyserver** [comment]
Closed in favour of https://github.com/moodlehq/moodleapp/pull/4248

**Pull request closed without merging**